content/en/tutorials/iam-policy-stream/index.md

```diff
@@ -0,0 +1,243 @@
+
+{{< command >}}
+$ awslocal sqs receive-message \
+    --queue-url http://sqs.us-east-1.localhost.localstack.cloud:4566/000000000000/s3-event-notification-queue
+{{< /command >}}
+
+The following output will show the message sent by the S3 bucket:
+
+```shell
+{
+    "Messages": [
+        {
+            "MessageId": "7c9d6b22-cb35-4a66-98dc-6f48dfc78f33",
+            "ReceiptHandle": "MTM4ZTg2NTYtMGIwNC00ZWE2LWIyM2EtNWNlZTIyOTZmOGE1IGFybjphd3M6c3FzOnVzLWVhc3QtMTowMDAwMDAwMDAwMDA6czMtZXZlbnQtbm90aWZpY2F0aW9uLXF1ZXVlIDdjOWQ2YjIyLWNiMzUtNGE2Ni05OGRjLTZmNDhkZmM3OGYzMyAxNzIwNTAzNjEyLjU2NDEyOTQ=",
+            "MD5OfBody": "10eacb105ec11badc56f7e0198e0c4ad",
+            "Body": "{\"Service\": \"Amazon S3\", \"Event\": \"s3:TestEvent\", \"Time\": \"2024-07-09T05:29:55.923Z\", \"Bucket\": \"s3-event-notification-bucket\", \"RequestId\": \"bfa882c0-a3b0-4549-b4c5-ac34167b3076\", \"HostId\": \"eftixk72aD6Ap51TnqcoF8eFidJG9Z/2\"}"
+        }
+    ]
+}
+```
+
+You can now check the IAM Policy Stream dashboard to confirm that there are no violations and that your AWS API requests have been successfully executed with the correct IAM policies.
+
+<img src="iam-policy-stream-no-violations.png" alt="IAM Policy Stream showcasing no violations" title="IAM Policy Stream showcasing no violations" width="900" />
+<br><br>
+
+### Generate a comprehensive policy
+
+In scenarios where there are many AWS services, and every AWS API request generates a policy it might be cumbersome to analyze every policy. In such cases, you can generate one comprehensive policy for all your AWS resources together.
+
+You can navigate to the **Summary Policy** tab on the IAM Policy Stream dashboard. This concatenates the policy per principle which the policy should be attached to. For the example above, you would be able to see the **Identity Policy** for the root user which has all the actions and resources inside one single policy file for the operations we performed.
+
+<img src="require-identity-based-policy.png" alt="Required identity based policy" title="Required identity based policy" width="900" />
+<br><br>
+
+On the other hand, you have the **Resource Policy** for the SQS queue, where you can see the permission necessary for the subscription. For larger AWS applications, you would be able to find multiple roles and multiple resource-based policies depending on your scenario.
+
+<img src="require-resource-based-policy.png" alt="Required resource based policy" title="Required resource based policy" width="900" />
+<br><br>
+
+## Conclusion
+
+IAM Policy Stream streamlines your development process by minimizing the manual creation of policies and confirming the necessity of granted permissions. However, it is advisable to manually confirm that your policy aligns with your intended actions. Your code may unintentionally make requests, and LocalStack considers all requests made during policy generation as valid.
```

Comment:
Maybe also emphasize that their test might not be a full picture, and requests not being made will also not be included in the policies.